specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,2958 @@
+      "properties": {
+        "resourceId": {
+          "description": "fully qualified resource Id e.g subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}",
+          "type": "string"
+        },
+        "resourceName": {
+          "description": "Azure resource name, required while creating the app component.",
+          "type": "string"
+        },
+        "resourceType": {
+          "description": "Azure resource type, required while creating the app component.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Azure resource display name",
+          "type": "string"
+        },
+        "resourceGroup": {
+          "description": "Resource group name of the Azure resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "description": "Subscription Id of the Azure resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "kind": {
+          "description": "Kind of Azure resource type",
+          "type": "string"
+        }
+      }
+    },
+    "AppComponentsMap": {
+      "description": "App Components model",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "Azure Load Testing resource Id",
+          "type": "string",
+          "readOnly": true
+        },
+        "testId": {
+          "description": "[Required, if testRunId is not given] Load test unique name",
+          "type": "string"
+        },
+        "testRunId": {
+          "description": "[Required if testId is not given] Load test run unique name",
+          "type": "string"
+        },
+        "name": {
+          "description": "AppComponent name",
+          "type": "string",
+          "readOnly": true
+        },
+        "value": {
+          "description": "AppComponents collection { resource id (fully qualified resource Id e.g subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}) : resource object } ",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/AppComponent"
+          }
+        }
+      }
+    },
+    "CertificateMetadata20221101": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The value of the certificate for respective type",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of certificate",
+          "enum": [
+            "AKV_CERT_URI"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CertificateType",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "description": "Name of the certificate.",
+          "type": "string"
+        }
+      }
+    },
+    "DefaultServerMetricsConfigListModel": {
```

Comment:
Please drop the "Model" suffix from all your schema names.

Reply:
removed